### PR TITLE
LPS-190903 [WIP] Allow users to edit account restriction field after creation

### PR DESCRIPTION
**Pull request by joseabelenda**
LPS-190903
After discussing some use cases, we identified that the fact that we don’t allow the update of this field have more negative consequences than allowing.
Users having the permission to update, but not to add an entry is a very edge case.
While, use cases where users need to change this field are more common.
For that reason, we decided to change the behavior to allow users to edit the account restriction field.

### CODE REVIEW

**Comment by joseabelenda** [comment]
@MarinhoFeliphe FYI

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -4225,16 +4225,6 @@ else if (StringUtil.equals(
...
-
-			Map<String, Serializable> originalValues = objectEntry.getValues();
-
-			if (!Objects.equals(
-					GetterUtil.getLong(originalValues.get(entry.getKey())),
-					GetterUtil.getLong(entry.getValue()))) {
-
-				throw new ObjectEntryValuesException.
-					UnmodifiableAccountEntryObjectField(objectField.getName());
-			}
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:4237]: hey @joseabelenda could you please remove this test [here](https://github.com/liferay/liferay-portal/blob/d2b300c6bb680325910b3f612b5cd30576c82384/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java#L2397-L2404)?

Thank you!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by joseabelenda** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
Resent here: https://github.com/liferay-objects/liferay-portal/pull/2530

Thanks for your contribution!!

cc: @joseabelenda

**Pull request closed without merging**